[CS2113T-F12-2] Get Jack’D

jonathanmui4 opened the pull request:
Get Jack'D helps fitness enthusiasts manage their workouts and fitness routines. It is optimized for CLI users to allow for quick planning of workouts and fitness routines through typing in commands.

qhokkohq committed:
Update text-ui-test

qhokkohq committed:
Get rid of stuff I added while experimenting.

IshaaanVyas committed:
Made some changes to DataClassDiagram and StorageDiagram. Also added BasicArchitectureDiagram into the DG.

qqkoh committed:
Merge pull request #129 from qqkoh/ImproveUI

Improve UI

JMattChiam committed:
Merge pull request #130 from IshaaanVyas/MakeSomeChangesToUiParserAndCommandUML

Add Basic Architecture UML Diagram

qhokkohq committed:
UI uml diagram.

KishorKumar11 committed:
Merge pull request #134 from qqkoh/UML-UI

UI uml diagram.

JMattChiam committed:
Add parser to Developer guide

JMattChiam committed:
Merge pull request #139 from JMattChiam/Update_DG

Add parser to Developer guide

KishorKumar11 committed:
Add command component to DG

KishorKumar11 committed:
Resolve merge conflict

KishorKumar11 committed:
Merge pull request #140 from KishorKumar11/Add-command-component-DG

Add Command component in Developer Guide

qhokkohq committed:
Add UI for DG.

qhokkohq committed:
Fix AddExerciseParser bug.

qqkoh committed:
Merge pull request #141 from qqkoh/master

Add UI DG

KishorKumar11 committed:
Add more workouts and include uniques sets and reps

qqkoh committed:
Merge pull request #142 from KishorKumar11/Modify-recommend-feature

Add more workouts and include unique sets and reps

qhokkohq committed:
Change command result to have ArrayList<?> and Map<String, ArrayList<?>>. Fix printTable and printMap.

qqkoh committed:
Merge pull request #143 from qqkoh/FixCommandResult

Change command result to have ArrayList<?> and Map<String, ArrayList<…

qhokkohq committed:
Refactor printTable in UI

qqkoh committed:
Merge pull request #144 from qqkoh/master

Refactor printTable in UI

qhokkohq committed:
Refactor search command

KishorKumar11 committed:
Resolved minor bug regarding recommend parameter

KishorKumar11 committed:
Add more JavaDoc for recommend class

KishorKumar11 committed:
Fix bug in Recommend Parser and Update Ui test

KishorKumar11 committed:
Add JunitTests for RecommendParser and RecommendCommand

KishorKumar11 committed:
Modified Ui test

KishorKumar11 committed:
Merge pull request #145 from KishorKumar11/Add-JunitTest-Recommend

Add Junit Test for Recommend

KishorKumar11 committed:
Implement edit feature

KishorKumar11 committed:
Update Ui test

KishorKumar11 committed:
Add JavaDoc

KishorKumar11 committed:
Add assertions and loggings

KishorKumar11 committed:
Add Junit tests

KishorKumar11 committed:
Add more Junit tests

KishorKumar11 committed:
Debug Ui test

KishorKumar11 committed:
Merge pull request #147 from KishorKumar11/Add-edit-feature

Add Edit feature

KishorKumar11 committed:
Update UG

KishorKumar11 committed:
Update DG

KishorKumar11 committed:
Merge pull request #150 from KishorKumar11/Add-edit-UG-DG

Add edit components in UG and DG

KishorKumar11 committed:
Update JavaDoc for command classes

KishorKumar11 committed:
Merge pull request #151 from KishorKumar11/Update-JavaDoc-Command-classes

Update JavaDoc for Command classes

IshaaanVyas committed:
Updated architecture diagram

IshaaanVyas committed:
some more changes to architecture uml

IshaaanVyas committed:
Added sequence diagram for basic architecture and convertAllWorkoutsToStorageModel()

IshaaanVyas committed:
Merge pull request #154 from IshaaanVyas/polishDG

Added Sequence Diagrams

qqkoh committed:
Merge branch 'master' into master

qqkoh committed:
Merge pull request #1 from AY2122S1-CS2113T-F12-2/master

Merge

IshaaanVyas committed:
made recommended exercises store inside file

qhokkohq committed:
Move help message from ui to helpCommand

qqkoh committed:
Merge pull request #155 from qqkoh/master

Move helpMessage from ui to helpCommand

IshaaanVyas committed:
Added a space

IshaaanVyas committed:
Merge remote-tracking branch 'upstream/master' into AddRecommendedWorkoutsIntoFile

IshaaanVyas committed:
updated text ui testing

IshaaanVyas committed:
try fix

IshaaanVyas committed:
cannot test recommended

jonathanmui4 committed:
Make deadlines to workouts optional

jonathanmui4 committed:
Add JUnit test for Deadline Workouts

jonathanmui4 committed:
Update text-ui-test

jonathanmui4 committed:
Merge pull request #156 from IshaaanVyas/AddRecommendedWorkoutsIntoFile

Add recommended workouts into file

jonathanmui4 committed:
Fix Indentation error

qhokkohq committed:
Add JavaDoc for SearchCommand, Ui, CommandResult.

qhokkohq committed:
gradlew check

qqkoh committed:
Merge pull request #163 from qqkoh/master

Add JavaDoc for SearchCommand, Ui, ComandResult.

jonathanmui4 committed:
Resolve Merge conflict in Workout class

jonathanmui4 committed:
Add new text-ui tests

jonathanmui4 committed:
Merge pull request #158 from jonathanmui4/feature-OptionalWorkoutDeadline

Make deadlines to created workouts optional

jonathanmui4 committed:
Update UG for create command

jonathanmui4 committed:
Update class diagram for Deadline Workout

jonathanmui4 committed:
Refactor lists and exercises into data package

jonathanmui4 committed:
Update DG with new user stories

jonathanmui4 committed:
Merge pull request #167 from jonathanmui4/documentation

Update documentation for create workout command

jonathanmui4 committed:
Refactor list package to data in test folder

jonathanmui4 committed:
Merge pull request #169 from jonathanmui4/master

Refactor list package to data in test folder

JMattChiam committed:
Add JUnit tests for CreateWorkoutParser

JMattChiam committed:
Add JUnit test for AddExerciseParser

Move generateCommand test for addExercise to addExerciseParser

JMattChiam committed:
Add JUnit test for MarkExerciseAsDoneParser

JMattChiam committed:
no message

JMattChiam committed:
Add JUnit test for DisplayExerciseParser

Same JUnit test also applies to the RemoveExerciseParser as the same
logic applies to both classes.

JMattChiam committed:
Add JUnit test for EditExerciseParser

JMattChiam committed:
Add JUnit test for SearchParser

jonathanmui4 committed:
Add automatic sorting of workouts according to deadline

jonathanmui4 committed:
Update text-ui-test

jonathanmui4 committed:
Add Junit Test for sortWorkout

jonathanmui4 committed:
Merge pull request #171 from jonathanmui4/feature-OptionalWorkoutDeadline

Add automatic sorting of workouts according to deadline

jonathanmui4 committed:
Update user story on DG

jonathanmui4 committed:
Merge pull request #172 from jonathanmui4/documentation

Update user story on DG

qhokkohq committed:
Allow searching by date

qhokkohq committed:
Solve arrayList warning in recommendWorkoutCommand

qqkoh committed:
Merge pull request #173 from qqkoh/SearchDeadline

Search deadline

qhokkohq committed:
Add JUnit tests for searchCommand

qqkoh committed:
Merge pull request #174 from qqkoh/JUnit-Search

Add JUnit tests for searchCommand

JMattChiam committed:
Change assertTrue to assertEquals

Try and see if the gradlew check on github is solved

JMattChiam committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F12-2/tp into Add_JUnit

JMattChiam committed:
Comment out Assertion in line 144 Parser

Hopefully fixes github test fail

JMattChiam committed:
Change assertTrue to assertEquals on DisplayExerciseParserTest

JMattChiam committed:
Add a BeforeEach which sets workoutMode to 0 for DisplayParserTest

crossed fingers

JMattChiam committed:
Add workoutMode = 0

JMattChiam committed:
Merge pull request #170 from JMattChiam/Add_JUnit

Add JUnit for parser child classes

IshaaanVyas committed:
Fixed edit bug and added Junits

KishorKumar11 committed:
Set theme jekyll-theme-merlot

IshaaanVyas committed:
Merge pull request #178 from IshaaanVyas/Add_JavaDoc_Junit

Fixed edit bug and added Junits

IshaaanVyas committed:
Fixed bug in enter command where workout index was getting ignored when inside a workout, also added Junit tests for this.

IshaaanVyas committed:
Added some more junits

KishorKumar11 committed:
Merge pull request #179 from IshaaanVyas/FixBugInEnterCommand

Fixed bug in enter command where workout index was getting ignored wh…

KishorKumar11 committed:
Implement clear feature

KishorKumar11 committed:
Add JavaDoc

KishorKumar11 committed:
Add assertions and loggings

KishorKumar11 committed:
Update help message

KishorKumar11 committed:
Update UG

KishorKumar11 committed:
Update DG

KishorKumar11 committed:
Update Ui-test

KishorKumar11 committed:
Merge pull request #181 from KishorKumar11/Add-clear-feature

Add clear feature

qhokkohq committed:
Ensure that when we are in workout mode and we delete the workout that we are in, workoutMode returns to 0.
Update help messages for when in WorkoutMode. Change table in User Guide.

qhokkohq committed:
gradlew check

qhokkohq committed:
Update text ui test

JMattChiam committed:
Add JUnit tests for SearchParser

qqkoh committed:
Merge pull request #188 from qqkoh/master

Update help messages for when in workout mode

JMattChiam committed:
Merge pull request #189 from JMattChiam/Add_JUnit

Add JUnit tests for SearchParser

KishorKumar11 committed:
Set theme jekyll-theme-cayman

KishorKumar11 committed:
Add author tags

KishorKumar11 committed:
Update EXPECTED.TXT

KishorKumar11 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F12-2/tp

KishorKumar11 committed:
Merge pull request #190 from KishorKumar11/master

Add author tags

KishorKumar11 committed:
Update README.md

AayushMathur7 commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,225 @@
+**`showResultToUser(CommandResult)`
+
+
+### Data Component
```
Maybe you could number your headings to make them clearer and cohesive? For example, 2.0, 2.1 etc.

AayushMathur7 commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,225 @@
+
+Location: `seedu.duke.ui`
+
+![img.png](umldg/UI.png)
```
Should the circles from the UI component diagram be removed to standardize with previous diagrams?

AayushMathur7 commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,225 @@
+### Value proposition: 
+Manage fitness routines quickly
 
 ## User Stories
```
Maybe you could condense the types of users even more? For example, bring the busy, forgetful, fit user under one umbrella so that you can provide more value to one specific type of user rather. Just a suggestion!

AayushMathur7 commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,225 @@
 # Developer Guide
```
Maybe include a "First time setup" section for the new users who aren't familiar with setting up?

AayushMathur7 reviewed (commented):
Overall, love the idea - I see myself using it :)

lamkwjonathan commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,225 @@
+
+![img.png](umldg/BasicArchitecture.png)
+
+__Note:__ Lines in this diagram denote that the component has references to most classes inside the component the arrow
```
It's good that you've explained the meaning of the arrows since this is a non-standard diagram. It helped make things clearer for me.

lamkwjonathan commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,225 @@
+* `reps` corresponding to the number of times the exercise is done in a given set.
+* `sets` corresponding to the number of times the whole exercise is repeated.
+
+__Storage.models__: `addToWorkoutListModel(WorkoutModel workout)` and `addToWorkoutModel(ExerciseModel exercise)` from the `storage.models` component
```
It was slightly confusing for me that a large explanation relating to Storage was made under the Data Component, especially when Storage and its function had yet to be fully explained. Perhaps reordering the explanations of Data Component and Storage Component would help.

lamkwjonathan commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,225 @@
+
+Location : `seedu.duke.command`
 
+![img.png](umldg/CommandDiagram.png)
```
The wordings in this class diagram are a little small, maybe consider increasing the font size or the image size?
![image](https://user-images.githubusercontent.com/77738460/139187299-2bed2a33-1311-4418-8b7f-447cd182c19c.png)

lamkwjonathan commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,225 @@
+__Note:__
+
+* `Command` : Deals with executing user commands on `workout` and `workoutList`.
+* `XYZExerciseCommand` are child classes which inherit from the `Command` class and implement/execute exercise related
```
Maybe it would be clearer to also explain that XYZ is a placeholder for the commands listed below.

lamkwjonathan commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,225 @@
+### CommandManager Component
+Location: `seedu.duke.parser`
+
+![img.png](umldg/CommandManager.png)
```
Perhaps the multiplicity can be placed on the opposite side of the arrow for clarity.
![image](https://user-images.githubusercontent.com/77738460/139188238-399146c9-7ccf-4a42-9695-7dc3b73b1cd3.png)

lamkwjonathan reviewed (commented):
Overall well done! Looks like it's almost complete as well.

asdzxc270697 commented on docs/umldg/DataSequenceDiagram.puml:
```diff
@@ -0,0 +1,39 @@
+    loop for all exercises
+        Workout -> Exercise ++ #Pink : convertToExerciseStorageModel(WorkoutModel)
+        Exercise -> ExerciseModel ** 
+        activate ExerciseModel #LightGreen
```
Maybe can change to loop condition to "all workouts" and "all exercises" don't need the for 
<img width="379" alt="Screenshot 2021-10-28 at 2 38 09 PM" src="https://user-images.githubusercontent.com/48401155/139200657-f96bc1d4-88d6-470e-a248-857f765458ba.png">

asdzxc270697 commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,225 @@
+- has a need to plan and remember their workouts quickly
+- Prefer desktop apps over other types
+- Can type fast
+- Comfortable using CLI apps
```
Would be good to start all points with capital letter
<img width="553" alt="Screenshot 2021-10-28 at 2 48 55 PM" src="https://user-images.githubusercontent.com/48401155/139202123-592afcec-5de9-419c-81f3-43d34706ef9e.png">

asdzxc270697 commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,225 @@
+  - [Storage Component](#storage-component)
+  - [Command Component](#command-component)
+  - [Command Manager Component](#commandmanager-component)
+- [Product Scope](#product-scope)
```
Would be good to add in all the components in the content page as listed in "Basic Architecture"
<img width="749" alt="Screenshot 2021-10-28 at 2 53 47 PM" src="https://user-images.githubusercontent.com/48401155/139203413-dd461e3d-0b90-44a7-a95d-dc8f1f35bcc4.png">

asdzxc270697 commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,225 @@
+4. `Ui` acknowledges the command and shows the result of the Command if any.
+
+Below is a high-level sequence diagram showing the `create` Command
+
+![img.png](umldg/BasicArchitectureSequenceDiagram.png)
+
+**`showResultToUser(CommandResult)`
+
```
I think maybe can use other special characters to represent showResultToUser(CommandResult) instead of asterisks, looks like it was multiplicity
<img width="311" alt="Screenshot 2021-10-28 at 3 02 30 PM" src="https://user-images.githubusercontent.com/48401155/139203895-18ddfd19-c455-4003-9122-ac3940570e43.png">

asdzxc270697 reviewed (commented):
Overall looks good! Looks like it's almost completed!

Demonshaha reviewed (commented):
The diagrams are very nice and concrete, but some of them can be more concise.

IshaaanVyas committed:
Added PPP and modified DG

KishorKumar11 committed:
Merge pull request #192 from IshaaanVyas/AddPPP_ModifyDG

Added PPP and modified DG

KishorKumar11 committed:
Add PPP

jonathanmui4 committed:
Add PPP and fix DG bug

jonathanmui4 committed:
Merge pull request #241 from jonathanmui4/master

Add PPP and fix DG bug

qhokkohq committed:
Fix search command "finish by" should not be included in searching result.
Fixes #223

qqkoh committed:
Merge pull request #242 from qqkoh/master

Fix search command "finish by" should not be included in searching result

qhokkohq committed:
Search within specified workout when in workout mode.

qhokkohq committed:
Add clarification in Search JavaDoc

KishorKumar11 committed:
Merge pull request #225 from KishorKumar11/AddPPP

Add PPP

qhokkohq committed:
Edit search test

qqkoh committed:
Merge pull request #243 from qqkoh/master

Search within specified workout when in workout mode.

JMattChiam committed:
Add PPP

JMattChiam committed:
to lower case

JMattChiam committed:
Add comment for MarkAsDoneParser

JMattChiam committed:
Merge pull request #249 from JMattChiam/master

Add PPP, add comment from Code review

JMattChiam committed:
Add using this guide section for UG

JMattChiam committed:
Merge pull request #251 from JMattChiam/Update_UserGuide

Add using this guide section for UG

qhokkohq committed:
Update display exercises command display message.

qhokkohq committed:
Merge branch 'master' of https://github.com/qqkoh/tp

qqkoh committed:
Merge pull request #252 from qqkoh/master

Update display exercises command display message.

JMattChiam committed:
Add reposense link

JMattChiam committed:
Merge pull request #253 from JMattChiam/master

Add reposense link

KishorKumar11 committed:
Solve PED issue - need of comma between params

KishorKumar11 committed:
Solve PED issue - clear function

KishorKumar11 committed:
Solve PED issue - recommend function arguments

KishorKumar11 committed:
Merge pull request #254 from KishorKumar11/Solve-PED-Issues

Solve PED issues

qhokkohq committed:
Add PPP, fix search by date bug.

KishorKumar11 committed:
Update PPP

qhokkohq committed:
Edit search command test.

qqkoh committed:
Merge pull request #255 from qqkoh/master

Add PPP, edit search by date bug

KishorKumar11 committed:
Solve TA comments

KishorKumar11 committed:
Remove unnecessary image files

KishorKumar11 committed:
Add pic in AboutUs.md

KishorKumar11 committed:
Merge pull request #256 from KishorKumar11/Update-PPP

Update PPP

KishorKumar11 committed:
Update AboutUs.md

jonathanmui4 committed:
Fix negative sets and reps in edit

jonathanmui4 committed:
Add exception messages to storage class

jonathanmui4 committed:
Add auto sorting feature in UG

JMattChiam committed:
Add invalid date catch

Added specific catch and message for when a user inputs an invalid date
for create command.

jonathanmui4 committed:
Close app upon error in storage

jonathanmui4 committed:
Fix gradle issues

JMattChiam committed:
Fix white space error for sets and reps

Fixed a bug that caused an invalid command when there were additional
spaces in the format.

jonathanmui4 committed:
Add javadoc to private methods in storage

jonathanmui4 committed:
Merge pull request #257 from jonathanmui4/debugging

Debugging for v2.1

JMattChiam committed:
Update UG to mention that recommend adds the workouts

JMattChiam committed:
Add help messages for help, search, enter and back commands.

Some minor edits to MESSAGE_USAGE for the commands as well

JMattChiam committed:
Merge pull request #258 from JMattChiam/Fix_PED_bugs

Fix ped bugs

qhokkohq committed:
Change date format to d MMM yyyy for search, revert back to search all workouts even when in workout mode.

qqkoh committed:
Merge branch 'master' into master

qqkoh committed:
Merge pull request #2 from AY2122S1-CS2113T-F12-2/master

pr

qqkoh committed:
Merge pull request #260 from qqkoh/master

Allow d MMM yyyy for search

qhokkohq committed:
Update UG for workout mode format for each command.

qhokkohq committed:
Edit display message of commands to include workout index, edit ug, edit search workout mode.

qqkoh committed:
Merge pull request #261 from qqkoh/master

Update UG for workout mode format for each command.

jonathanmui4 committed:
Update Guides

jonathanmui4 committed:
Update Manual Testing Instructions for DG

jonathanmui4 committed:
Merge pull request #262 from jonathanmui4/documentation

Update Documentation

IshaaanVyas committed:
Added expected outcomes for edit, clear and display when inside a workout. Fixes #206 , #232 and #237

IshaaanVyas committed:
Added display and mark as done in the enter command tip

IshaaanVyas committed:
fixed bug where it was possible to create workout with no name and a deadline

IshaaanVyas committed:
added junit tests for bug fix

IshaaanVyas committed:
Changed PPP to proper format

KishorKumar11 committed:
Update introduction and reformat notes and warnings

KishorKumar11 committed:
Update clear section of UG

KishorKumar11 committed:
Update appendix of DG

KishorKumar11 committed:
Update PPP

KishorKumar11 committed:
Merge pull request #263 from IshaaanVyas/fix_dryRun_bugs

Fix dry run bugs

KishorKumar11 committed:
Merge branch 'master' into Update-UGDG

KishorKumar11 committed:
Merge pull request #264 from KishorKumar11/Update-UGDG

Update UG and DG

KishorKumar11 committed:
Update PPP 

Exceeded above 2 pages when tried to save it as PDF

JMattChiam committed:
Clear command from entered workout will return user to main view

JMattChiam committed:
Merge pull request #267 from JMattChiam/master

Clear command from entered workout will return user to main view

IshaaanVyas committed:
Fixed bug which showed exception when a file called data was in the same folder as getjackd

IshaaanVyas committed:
Added documentation guide into DG and added manual testing for enter command

JMattChiam committed:
Polish and standardise formatting of User Guide

JMattChiam committed:
Update Version and recency of GetJackD.java

IshaaanVyas committed:
Merge pull request #269 from IshaaanVyas/FixBugWhenFileCalledDataInsideFolder

Fixed bug which showed exception when a file called data was in the s…

IshaaanVyas committed:
Merge pull request #270 from IshaaanVyas/ApplyFinishingTouchesToUGDG

Added documentation guide into DG and added manual testing for enter …

JMattChiam committed:
Minor fixes

JMattChiam committed:
Merge pull request #271 from JMattChiam/Update_UserGuide

Update user guide

KishorKumar11 committed:
Fixed anchor problem for logging and documentation guide

KishorKumar11 committed:
Add Setting up and Getting started component in DG

KishorKumar11 committed:
Update PPP

IshaaanVyas committed:
Merge pull request #273 from KishorKumar11/Update-UML-Command-Component

Fixed anchor problem for logging and documentation guide

IshaaanVyas committed:
Added troubleshooting guide into UG

qhokkohq committed:
Update DG with Manual Testing

qqkoh committed:
Merge pull request #276 from qqkoh/master

Update DG with Manual Testing

IshaaanVyas committed:
Merge pull request #274 from IshaaanVyas/Add_troubleshootingHelpInUG

Added troubleshooting guide into UG

IshaaanVyas committed:
Added comment for parseWorkoutAndExerciseIndex inside Parser.java

IshaaanVyas committed:
Merge pull request #278 from IshaaanVyas/master

Added comment for parseWorkoutAndExerciseIndex inside Parser.java

qhokkohq committed:
Add comments for private methods in Command Tests.

qqkoh committed:
Merge pull request #279 from qqkoh/master

Add comments for private methods in Command Tests.

qhokkohq committed:
Add profile pic

qhokkohq committed:
Add page breaks

qqkoh committed:
Merge pull request #280 from qqkoh/master

Add profile pic

qhokkohq committed:
update for pdf format

qqkoh committed:
Merge pull request #281 from qqkoh/master

update for pdf format

qhokkohq committed:
update ug for pdf formatting

qqkoh committed:
Merge pull request #282 from qqkoh/master

update ug for pdf formatting

qhokkohq committed:
again

qqkoh committed:
Merge pull request #283 from qqkoh/master

ug pdf formatting again

qhokkohq committed:
correct date format to d mmm yyyy

qqkoh committed:
Merge pull request #284 from qqkoh/master

correct date format to d mmm yyyy

IshaaanVyas committed:
fixed small typo in UG

IshaaanVyas committed:
Merge pull request #285 from IshaaanVyas/master

fixed small typo in UG

KishorKumar11 committed:
Update AboutUs.md

Added links to the picture and GitHub

JMattChiam committed:
Edit PPP

JMattChiam committed:
Merge pull request #286 from JMattChiam/master

Edit PPP

JMattChiam committed:
Add pic for AboutUs

JMattChiam committed:
Merge pull request #287 from JMattChiam/master

Add pic for AboutUs

jonathanmui4 committed:
Update PPP

jonathanmui4 committed:
Merge pull request #288 from jonathanmui4/documentation

Update PPP

qhokkohq committed:
include date example in ug.

qqkoh committed:
Merge pull request #289 from qqkoh/master

include date example in ug.

KishorKumar11 committed:
Add boldings and emojis

KishorKumar11 committed:
Merge pull request #290 from KishorKumar11/Update-UG-Design

Add bolds and emojis to UG

KishorKumar11 committed:
Update anchor links

KishorKumar11 committed:
Merge branch 'master' into Fix-anchor-links-attempt1

KishorKumar11 committed:
Merge pull request #291 from KishorKumar11/Fix-anchor-links-attempt1

Update anchor links

KishorKumar11 committed:
Update anchor links

KishorKumar11 committed:
Merge pull request #292 from KishorKumar11/Fix-anchor-links-attempt2

Update anchor links

KishorKumar11 committed:
Update anchor links

KishorKumar11 committed:
Merge pull request #293 from KishorKumar11/Fix-anchor-links-attempt3

Update anchor links

KishorKumar11 committed:
Update anchor links

KishorKumar11 committed:
Merge pull request #294 from KishorKumar11/Fix-anchor-links-attempt4

Update anchor links
